scripts/pages/pgFavorites.ts

```diff
@@ -34,11 +36,16 @@ export default class PgFavorites extends withDismissAndBackButton(PgFavoritesDes
     }
     addToCartSelectedProducts() {
         this.flCartCheckout.btnCartCheckout.onPress = () => {
-            this.selectedProducts.forEach((product, index) => {
-                store.dispatch(storeActions.AddToBasket({ product, count: 1 }));
-                store.dispatch(storeActions.RemoveFromFavorites({ productId: product._id }));
-                this.selectedProducts.splice(index, 1);
+            let ids = [];
```

Comment:
We should use const if the variable will not change. @suleymanbasbug

Reply:
ok.